### DISCUSSION TITLE
deleted

### DISCUSSION THREAD

**Discussion by doobersn** [qdm12/gluetun#1285] [Q&A]
deleted

**Comment by qdm12** [comment]
Hello, do you still need help on this?

**Reply by doobersn** [reply]
> Hello, do you still need help on this?

I think I figured it out, thank you! I guess while you’re here I’ll ask: I’m using Mullvad with wiregaurd. Does the server change or is it always the same one? Also, I can confirm that the two containers (gluetun and qbit) have the same ip by running ifconfig right? I do see qbit has an ip of a vpn server when I run curl ifconfig.io. Thanks!

**Reply by qdm12** [reply]
> Does the server change or is it always the same one

This depends on your server filters. If you set `SERVER_COUNTRIES=Netherlands`, it will pick a server at random in Netherlands (and pick randomly on auto-heal as well, if it happens). You can pin it to a particular server using `SERVER_HOSTNAMES=somehostname.com` if you want, although that's less 'resilient' since if the server goes down it won't jump to another server in the same region/city.

> Also, I can confirm that the two containers (gluetun and qbit) have the same ip by running ifconfig right?

Sure. I usually do `docker exec mycontainer wget -qO- https://ipinfo.io` or `docker exec mycontainer curl -s https://ipinfo.io` :wink:

**Reply by doobersn** [reply]
Thank you so much! If i don’t specify a country does it just refresh at a certain interval? The country field was optional but i did have to set a country (on the mullvad website) to get the private key. However i could swear i’ve checked the location of the ip address that’s output when i do the curl ifconfig.io command and it changes and isn’t only in that country (again, vpn is mullvad). 

> > Does the server change or is it always the same one
> 
> This depends on your server filters. If you set `SERVER_COUNTRIES=Netherlands`, it will pick a server at random in Netherlands (and pick randomly on auto-heal as well, if it happens). You can pin it to a particular server using `SERVER_HOSTNAMES=somehostname.com` if you want, although that's less 'resilient' since if the server goes down it won't jump to another server in the same region/city.
> 
> > Also, I can confirm that the two containers (gluetun and qbit) have the same ip by running ifconfig right?
> 
> Sure. I usually do `docker exec mycontainer wget -qO- https://ipinfo.io` or `docker exec mycontainer curl -s https://ipinfo.io` 😉

**Reply by qdm12** [reply]
For Mullvad, the key is the same for all locations, so it's up to the server filters you set in Gluetun.
If you don't specify a country, it picks a server at random at start, and when the vpn gets restarted (due to unhealthy, or by the http control server - there is no periodic restart for now).
